gateway-ha/src/main/java/io/trino/gateway/ha/resource/GatewayWebAppResource.java

```diff
@@ -59,6 +69,7 @@
 import static java.util.Objects.requireNonNullElse;
 
 @Path("/webapp")
+@Singleton
```

Comment:
Why do need this change?

Reply:
Sorry not needed, forgot to remove it. I'll remove 👍